roxie/ccd/ccdqueue.cpp

```diff
@@ -1573,6 +1594,7 @@ class CRoxieWorker : public CInterface, implements IPooledThread
                 {
                     for (unsigned subChannel = 0; subChannel < mySubChannel; subChannel++)
                         noteNodeSick(header.subChannels[subChannel]);
+                    logctx.setStatistic(StTimeIBYTIDelay, (msTick()-start)*1000000);
```

Comment:
I keep thinking it is a shame we're not using nsTick().  Not a change to make at the same time.
Could use for StTimeAgentQueue, (which is also interesting on a sub-ms timing level).

Reply:
Can do...